driver/driver.go

```diff
@@ -83,6 +83,34 @@ func (d *ContrailDriver) StartServing() error {
 
 	log.Infoln("Started serving on ", pipeAddr)
 
+	rootNetwork, err := hns.GetHNSNetworkByName(common.RootNetworkName)
+	if err != nil {
+		return err
+	}
+	if rootNetwork == nil {
+
+		subnets := []hcsshim.Subnet{
+			{
+				AddressPrefix:  "0.0.0.0/24",
+				GatewayAddress: "0.0.0.0",
+			},
+		}
+		configuration := &hcsshim.HNSNetwork{
+			Name:               common.RootNetworkName,
+			Type:               "transparent",
+			NetworkAdapterName: d.networkAdapter,
+			Subnets:            subnets,
+		}
+		rootNetID, err := hns.CreateHNSNetwork(configuration)
```

Comment:
As we discussed IRL - creating HNS network after starting the go routine in https://github.com/codilime/contrail-windows-docker/pull/40/files#diff-ceffdf483743f1bed633959d8c4575f3R79 might cause a data race.